### PR TITLE
Audit logging

### PR DESCRIPTION
**Pull request by jekuaitk**
https://leantime.itkdev.dk/#/tickets/showTicket/2917

* Audit logging

### CODE REVIEW

File: src/Plugin/WebformElement/MineOrganisationsData.php [hunk 1]
```diff
@@ -875,4 +924,32 @@ private function setSearchInformation(string $brugerId): void {
     }
   }
 
+  /**
+   * Audit logs viewed data.
+   */
+  private function auditLog(string $data): void {
+
```
Comment (cableman) [inline src/Plugin/WebformElement/MineOrganisationsData.php]: Why the empty line

File: src/Plugin/WebformElement/MineOrganisationsData.php [hunk 2]
```diff
@@ -582,6 +624,16 @@ private function updateFunktionSubElements(&$element, string $funktionsId): void
...
   }
 
+  /**
+   * Get current user organisation bruger id.
+   */
+  private function getCurrentUserOrganisationUserId(): ?string {
+    // Let other modules set organisation user id.
+    $event = new OrganisationUserIdEvent();
+    $this->eventDispatcher->dispatch($event);
+    return $event->getUserId();
```
Comment (cableman) [inline src/Plugin/WebformElement/MineOrganisationsData.php:627]: Think that should be an empty line before return.

**Review by cableman — APPROVED** [review]

File: src/Plugin/WebformElement/MineOrganisationsData.php (hunk 1 above)
Comment (jekuaitk) [reply src/Plugin/WebformElement/MineOrganisationsData.php]: Fixed in 054a26d22026668510bf8936e038cdd78f05c336.

File: src/Plugin/WebformElement/MineOrganisationsData.php (hunk 2 above)
Comment (jekuaitk) [reply src/Plugin/WebformElement/MineOrganisationsData.php:627]: Fixed in 054a26d22026668510bf8936e038cdd78f05c336.

**Pull request merged**